### PR TITLE
SunEditor text editor

### PR DESCRIPTION
**Pull request by Enoch-Hsiao**
No account needed, free, and easy to configure.

Configurations Documentation:
[SunEditor Docs](https://github.com/JiHong88/SunEditor/blob/master/README.md#options)

Responsiveness:
TextEditor

TextEditor

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
-            'insertdatetime media table paste code help wordcount'
-          ],
-          toolbar:
-            'undo redo | formatselect | bold italic backcolor | \
-            alignleft aligncenter alignright alignjustify | \
-            bullist numlist outdent indent | removeformat | help'
+      <SunEditor 
+        setOptions={{
+          width:'100%',
+          height: 400,
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:23]: You probably want to do something more responsive than just hard coded 400px. Chrome lets you simulate a phone to test stuff like that. Not for this PR, but there are also some small issues with login and the drawer on mobile.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
+          placeholder: "Enter in introduction of component...",
+          buttonList: [
+            ['font', 'fontSize', 'formatBlock'],
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:26]: Similarly here, maybe check out some examples of how other sites handle this on mobile. Maybe only include the most important few options on mobile? Right now there is a ton of screen real estate being taken up by the buttons on mobile

**Review by powellsl99 — CHANGES_REQUESTED** [review]
Looks really great! Super professional looking! Just a few suggestions for mobile responsiveness

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js (hunk 1 above)
Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:23]: Can you share a screenshot of the "login" on mobile?

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js (hunk 2 above)
Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:26]: Yea I definitely change the options for mobile devices depending on screen width.

Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:26]: Actually there is a way to do it within the component :)

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js (hunk 1 above)
Comment (powellsl99) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:23]: Sorry, I actually meant the landing page 😅 Nothing major, just the top buttons could use some smaller text / better spacing. There is a library I like to use called react-device-detect I think that has a func called isMobile()
Screen Shot 2020-10-07 at 3 23 23 PM
Screen Shot 2020-10-07 at 3 24 02 PM

Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:23]: A responsive sidebar should be implemented in another PR, I definitely can adjust the buttons in this PR though.

Comment (powellsl99) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js:23]: Don't worry about either if you don't want for this PR, just wanted to mention it as I saw it!

**Review by powellsl99 — APPROVED** [review]
LGTM! Great job!

**Pull request merged**